File: Exercism Codes/Java/Darts.java
import java.lang.Math;
class Darts {
    int score(double xOfDart, double yOfDart) {
        double dist = Math.sqrt(xOfDart*xOfDart + yOfDart*yOfDart);
        if (dist > 10) return 0;
        else if (dist > 5) return 1;
        else if (dist > 1) return 5;
        else return 10;
    }
}

File: Exercism Codes/Visual Basic/Darts.vb
Imports System

Public Module Darts
    Public Function Score(ByVal x As Double, ByVal y As Double) As Integer
        Dim distance As Double = Math.Sqrt(x * x + y * y)
        If distance > 10 Then
            Return 0
        ElseIf distance > 5 Then
            Return 1
        ElseIf distance > 1 Then
            Return 5
        Else
            Return 10
        End If
    End Function
End Module
